parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -183,54 +200,35 @@ private void addRowGroup(ParquetMetadata parquetMetadata, List<RowGroup> rowGrou
     return converted;
   }
 
-  private static final class EncodingList {
```

Comment:
@julienledem  Just wanna make sure we don't miss any information after removing this class. What's the original intention of creating this EncodingList instead of using the Set directly